build/package/install-agent-spawn-host.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -ex
+
+while getopts 'i:h:t:u:v:' opt; do
+  case $opt in
+  i) keyfile="$OPTARG" ;; # SSH identity file
+  u) user="$OPTARG" ;; # Username on the remote host
+  h) hostsfile="$OPTARG" ;; # Output of Evergreen host.list
+  t) groupid="$OPTARG" ;; # APIGroupId
+  v) apiKey="$OPTARG" ;; # APIKey
+  *) exit 1 ;;
+  esac
+done
+
+declare -a ssh_opts
+ssh_opts[0]="-o"
+ssh_opts[1]="UserKnownHostsFile=/dev/null"
+ssh_opts[2]="-o"
+ssh_opts[3]="StrictHostKeyChecking=no"
+ssh_opts[4]="-q"
+ssh_opts[5]="-o"
+ssh_opts[6]="ConnectTimeout=30"
+ssh_opts[7]="-o"
+ssh_opts[8]="TCPKeepAlive=yes"
+ssh_opts[9]="-o"
+ssh_opts[10]="ServerAliveInterval=15"
+ssh_opts[11]="-o"
+ssh_opts[12]="ServerAliveCountMax=20"
+ssh_opts[13]="-i"
+ssh_opts[14]="$keyfile"
+
+hosts=$(
+  cat <<EOF | python - "$hostsfile"
+import sys
+import json
+with open(sys.argv[1]) as hostsfile:
+    hosts = json.load(hostsfile)
+    for host in hosts:
+        print(host["dns_name"])
+EOF
+)
+for host in $hosts; do
+  set +e
+  echo "Installing dependeces on $host"
+  ssh "${ssh_opts[@]}" -tt "$user@$host" 'bash -s' <<'ENDSSH'
+        sudo apt-get install -y --no-install-recommends ca-certificates curl logrotate openssl snmp && exit
+ENDSSH
+  echo "Installing the automation agent on $host"
+  ssh "${ssh_opts[@]}" -tt "$user@$host" ARG1="$groupid" ARG2="$apiKey" 'bash -s' <<'ENDSSH'
+        echo "Downloadind and extracting the automation agent"
+        curl -OL https://cloud-dev.mongodb.com/download/agent/automation/mongodb-mms-automation-agent-manager_latest_amd64.ubuntu1604.deb
```

Comment:
ok to use ` https://cloud-dev.mongodb.com` but we have an env variable for the base url don't we? could we use that one here as well so they match?

Reply:
As I said here: https://github.com/mongodb/mongocli/pull/179#discussion_r432089778, the piece of code that changes properties inside the `mongodb-mms/automation-agent.config` does not work if the string contains special characters (in this case it would be https://.....).
I have tried only a few attempts to make that piece of code working also with special characters. Is there any difference in downloading the latest agent from cloud-dev instead of cloud? It might not be worthed the try if is the same

Reply:
Sorry for the previous comment. I misunderstood it.